augur/clades.py

```diff
@@ -62,7 +62,7 @@ def read_in_clade_definitions(clade_file):
     df = pd.read_csv(
         clade_file,
         sep='\t' if clade_file.endswith('.tsv') else ',',
-        comment='#'
+        comment='#', na_filter=False
```

Comment:
Looks like the rsv run in CI [failed](https://github.com/nextstrain/augur/actions/runs/5976131009/job/16213388426#step:7:1256) because it relied on the previous behavior which allowed for nullable numeric values under `site`:

```
    clades[clade][(row.gene, int(row.site)-1)] = row.alt
ValueError: invalid literal for int() with base 10: ''
```

This should be fixable by properly specifying dtypes for the clade file columns.

I can do that after finishing some other work, unless you get to it before me.

Reply:
the root cause was probably something else. The way clades work is that we construct a graph of all clades obtained via `df.clades.unique()`. If there are empty lines in the file, there was now a clade `""`. And that clade had definitions that are just `""`.

Reply:
Huh, I was wondering why the default `skip_blank_lines=True` was working for all other pathogen clade TSVs but not RSV. Turns out the empty lines in [config/clades_genome_a.tsv](https://github.com/nextstrain/rsv/blob/534be6f85c60d3f82e497791ec58fa7226365a27/config/clades_genome_a.tsv) are tab separated empty values instead of just empty lines.

Reply:
@joverlee521 oh, that explains it!

@rneher is this file auto-generated? Maybe we can fix that file instead of working around it here... or would it be good to filter out empty clade names anyways (even if `gene`/`site`/`alt` are defined)?

Reply:
It's probably best to keep the empty clade name filter here. I can't think of where a clade would not be named. I think even the basal clade would be named `root` or something similar. 

I'll update the RSV clade TSVs just to keep it uniform with others...

Reply:
https://github.com/nextstrain/rsv/pull/35